[Term Entry] PHP Date function date_sub() (#2730)
Codecademy/docs: content/php/concepts/date-functions/terms/date-sub/date-sub.md

---
Title: 'date_sub()'
Description: 'Subtracts a specified interval from a given date.'
Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Time'
CatalogContent:
  - 'learn-php'
  - 'paths/computer-science'
---

The **`date_sub()`** function subtracts a number of days, months, years, hours, minutes and seconds from a given date.

## Syntax

```pseudo
date_sub($object, $interval)
```

The `date_sub()` function has two required parameters:

> - `$object`: This parameter specifies the DateTime object returned by date_create().

Reviewer: ```suggestion
- `$object`: This parameter specifies the `DateTime` object returned by `date_create()`.
```

Reviewer: generally we use backticks on specific terms, functions, symbols specific to the code shown or language. If it's a generic term such as arrays or classes I would omit the backticks.

> - `$interval`: This parameter specifies the DateInterval object which we want to subtract.

Reviewer: ```suggestion
- `$interval`: This parameter specifies the `DateInterval` object to be subtracted.
```

Reviewer: we avoid the use of first and second person pronouns (I, you, we) in docs

> The `date_sub()` function returns a DateTime object after subtracting the interval.

Reviewer: ```suggestion
The `date_sub()` function returns a `DateTime` object after subtracting the interval.
```

## Example

> The following example uses the `date_sub()` function to subtract 30 days from the 3 March, 2023:

Reviewer: ```suggestion
The following example uses the `date_sub()` function to subtract 30 days from the passed date:
```
